specification/TileFormats/PointCloud/README.md

```diff
@@ -325,6 +351,22 @@ var batchTableJSON = {
 };
 ```
 
+#### Draco compression
+
+```javascript
+var featureTableJSON = {
+    POINTS_LENGTH : 4,
+    DRACO : {
+        byteOffset : 0
+        byteLength : 36,
+        semantics : ['POSITION', 'RGB']
+    }
+};
+
+// Use Draco encoder to build dracoBuffer
+var featureTableBinary = dracoBuffer;
```

Comment:
I don't think this line in the example adds much. Instead I would say that the binary much be encoded with the Draco encoder in the section above and link to the encoder tool(s).